.github/workflows/agent-charm-release-edge.yml

```
@@ -0,0 +1,27 @@
+name: Release testflinger-agent charm to latest/edge
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - agent/testflinger-agent-charm/**
```

I'm wondering if the path should be like this. Please correct me if I misunderstood.
```suggestion
      - agent/charms/testflinger-agent-charm/**
```